doc/src/stdlib/file.md

```diff
@@ -50,6 +50,7 @@ Base.Filesystem.basename
 Base.@__FILE__
 Base.@__DIR__
 Base.@__LINE__
+Base.propagate_sourceloc!
```

Comment:
Correct, however this is most related to `@__LINE__`, `@__FILE__` and `@__DIR__`.  Perhaps all these should be moved to another location.  The location in the manual ("metaprogramming") seems much more sensible for these things, as they're all about introspection of the macro expansion process.